tutorials/create-events-dataset.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: Creating the Events Dataset
+---
+## Purpose
+
+This lesson shows you how to create a dataset suitable for storing standard event records, as defined in [The Unified Logging Infrastructure for Data Analytics at Twitter][paper]. You define a dataset schema, a partition strategy, and a URI that specifies the storage scheme.
```

Comment:
The modules stand alone. The structure is provided by a TOC. The skills description is fine.